files/start-strongswan.sh

```diff
@@ -22,9 +22,11 @@ _start_strongswan() {
     wait "$child"
 }
 
+# Set default mode
+: ${USE_ENV_CONFIG:=HIDDEN_PUBIP_HOST}
```

Comment:
i would rather get rid of the "hidden_pubip_host" mode. can we agree on example-sets of variables instead? this is not really a different "mode".